src/main/java/org/semver4j/internal/Comparator.java

```diff
@@ -105,6 +105,9 @@ private static int compareIdentifiers(@NotNull final String a, @NotNull final St
     private static Integer checkAlphanumericPrerelease(@NotNull final String a, @NotNull final String b) {
         String[] tokenArrA = a.split(TRAILING_DIGITS_EXTRACT);
         String[] tokenArrB = b.split(TRAILING_DIGITS_EXTRACT);
+        if (tokenArrA.length != tokenArrB.length) {
+            return tokenArrA.length - tokenArrB.length;
```

Comment:
Could you add an appropriate test for this case?

Reply:
Yep I was thinking about that but I didn't see this class in the test package so I didn't do any
But I'll definitely add that